src/actions/index.js

```diff
@@ -0,0 +1,52 @@
+import axios from "axios";
+import { SuccessIcon, DangerIcon } from "lucid-ui";
+
+export const actionTypes = {
+  SET_TODOS: "SET_TODOS",
+  SET_PAGE: "SET_PAGE",
+  SET_PAGE_SIZE: "SET_PAGE_SIZE",
+  SET_TODOS_LIST: "SET_TODOS_LIST",
+  SET_ERROR: "SET_ERROR",
+};
+
+export function getTodos() {
+  return async (dispatch) => {
+    await axios
```

Comment:
told you to create a util function in a separate file for api calls and not directly call here.